package/MDAnalysis/coordinates/base.py

```diff
@@ -1942,6 +1958,22 @@ def __init__(self, filename, convert_units=None, n_atoms=None, **kwargs):
         self._ts_kwargs = ts_kwargs
         self._read_first_frame()
 
+    def copy(self):
```

Comment:
it's a little ugly copying this method, but SingleFrameReader and ReaderBase both inherit from ProtoReader, which has a slightly different signature, so putting the `copy` method there (rightly) set off the linter

tbh we should probably get rid of SingleFrameReader and squash its logic into the ReaderBase...

Reply:
I'm for getting rid of `SingleFrameReader`

Reply:
but where is the signature different?

Reply:
these readers both inherit from `ProtoReader` which has a different signature to `ReaderBase` and `SingleEtc`